src/ir/enum_ty.rs

```diff
@@ -49,6 +54,11 @@ impl Enum {
         &self.variants
     }
 
+    /// The special attributes of enumeration
+    pub fn attributes(&self) -> &Vec<Attribute> {
```

Comment:
I have tried `clang_Cursor_getOffsetOfField`, it seems doesn't work.
```c
struct header
{
    char proto;
    unsigned int size __attribute__ ((packed));
    //unsigned char data[] __attribute__ ((aligned(8)));
} __attribute__ ((aligned, packed));
```
First, this API doesn't support incomplete type, if we uncomment the `data` field, all field return `CXTypeLayoutError_Incomplete`.

Second, the API return wrong offset, the following C code will generate
> sizeof(header) = 16
> offsetof(size) = 1
```c
#include <stdio.h>
#include <stdlib.h>
#include <stddef.h>

#include "layout.h"

int main() {
    struct header header = {0};

    printf("sizeof(header) = %lu\n", sizeof(struct header));
    printf("offsetof(size) = %lu\n", offsetof(struct header, size));
}
```
But the API return offset is 8
> DEBUG:bindgen::codegen: <Item as CodeGenerator>::codegen: self = Item { id: ItemId(1), local_id: Cell { value: None }, next_child_local_id: Cell { value: 1 }, canonical_name_cache: RefCell { value: None }, comment: None, annotations: Annotations { opaque: false, hide: false, use_instead_of: None, disallow_copy: false, private_fields: None, accessor_kind: None, constify_enum_variant: false }, parent_id: ItemId(0), kind: Type(Type { name: Some("header"), layout: Some(Layout { size: 16, align: 16, packed: false }), kind: Comp(CompInfo { kind: Struct, fields: [Field { name: Some("proto"), ty: ItemId(3), comment: None, annotations: Annotations { opaque: false, hide: false, use_instead_of: None, disallow_copy: false, private_fields: None, accessor_kind: None, constify_enum_variant: false }, bitfield: None, mutable: false, offset: 0 }, Field { name: Some("size"), ty: ItemId(5), comment: None, annotations: Annotations { opaque: false, hide: false, use_instead_of: None, disallow_copy: false, private_fields: None, accessor_kind: None, constify_enum_variant: false }, bitfield: None, mutable: false, offset: 8 }], template_args: [], methods: [], constructors: [], base_members: [], ref_template: None, inner_types: [], inner_vars: [], has_vtable: false, has_destructor: false, has_nonempty_base: false, has_non_type_template_params: false, packed: true, is_anonymous: false, found_unknown_attr: true, detect_derive_debug_cycle: Cell { value: false }, detect_has_destructor_cycle: Cell { value: false }, is_forward_declaration: false }), is_const: false }) }